### DISCUSSION TITLE
Erldb v2

### DISCUSSION THREAD

**Discussion by Taure** [erldb/erldb#50] [Ideas]
# version 2 of erldb

We are redoing how we want erldb to work and the first phase is to make it default to Postgresql first. We are also redefining the syntax around it.

https://github.com/erldb/erldb/tree/erldb_v2

**Comment by burbas** [comment]
Let's first discuss how we want models to work. My suggestion is that we have something similar to what we have today that a model translates to a record. I don't particularly like that we are using parameterized modules today, so let's skip that tbh. So erldb will have two parts; one that describes models and another that actually are handling all the database-specific queries.
